kolibri/core/assets/src/views/AppBar.vue

```diff
@@ -123,6 +124,7 @@
     },
     mixins: [navComponentsMixin, themeMixin],
     $trs: {
+      openNav: 'Open sidebar navigation',
```

Comment:
Maybe "Open site navigation", since for screen readers 'sidebar' is less meaningful?

Reply:
It was an attempt to distinguish the sidebar menu (left), from the user menu (right).